frontend-vue/src/router.js

```diff
@@ -1,23 +1,37 @@
 import Vue from 'vue'
 import Router from 'vue-router'
-import Home from './views/Home.vue'
+import Home_page from './views/Home_page.vue'
```

Comment:
IMO adding "page" to the end of the views is a bit redundant, as we already know that they are pages.

So that name changes I would suggest for these are
* `Home_page` -> `Home`
* `Output_page` -> `Char_output` (to keep consistent with `Char_input`)